test/meson.build

```diff
@@ -1,5 +1,12 @@
+compiler_flags_test = compiler_flags
+
 if lib_type == 'static'
-    compiler_flags += ['-DTVG_STATIC']
+    compiler_flags_test += ['-DTVG_STATIC']
```

Comment:
could you please use `test_compiler_flags` ?

Reply:
@hermet  Fixed and checked on macos